docs/architecture/microservices/microservice-ddd-cqrs-patterns/net-core-microservice-domain-model.md

```diff
@@ -12,7 +12,7 @@ In the previous section, the fundamental design principles and patterns for desi
 
 The folder organization used for the eShopOnContainers reference application demonstrates the DDD model for the application. You might find that a different folder organization more clearly communicates the design choices made for your application. As you can see in Figure 7-10, in the ordering domain model there are two aggregates, the order aggregate and the buyer aggregate. Each aggregate is a group of domain entities and value objects, although you could have an aggregate composed of a single domain entity (the aggregate root or root entity) as well.
 
-![The Solution Explorer view for the Ordering.Domain project, showing the AggregatesModel folder containing the BuyerAggregate and OrderAggregate folders, each one containing it's entity classes, value object files and so on. ](./media/image11.png)
+![Screenshot of the Ordering.Domain project in Solution Explorer.](./media/net-core-microservice-domain-model/ordering-microservice-container.png)
```

Comment:
This image can also be considered "complex" so it's better to use the new image tag as mentioned before:

```markdown
:::image type="complex" source="<folderPath>" alt-text="<alt-text>":::
<long description here>
:::image-end:::
```